rudder-core/src/main/scala/com/normation/rudder/services/policies/SystemVariableService.scala

```diff
@@ -336,19 +336,43 @@ class SystemVariableServiceImpl(
     // else it's only the policy server
     val policyServerVars = if (nodeInfo.isPolicyServer) {
 
+      val childrenByPolicyServer = allNodeInfos.values.toList.groupBy( _.policyServerId )
```

Comment:
Maybe you could put that value in a parent; or you will have to build it for each node